certified-connectors/Tikit/apiProperties.json

```diff
@@ -0,0 +1,51 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "b13c40ee-e073-459e-96b5-3f3cca046a37",
+          "scopes": [],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "b13c40ee-e073-459e-96b5-3f3cca046a37",
+            "IsOnbehalfofLoginSupported": true
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "b13c40ee-e073-459e-96b5-3f3cca046a37"

```

Comment:
@TikitTeam Please remove the secret value as per Microsoft's security policy. You can replace it with dummy text. "[[DUMMY]]"

Reply:
fixex